test/parallel/test-process-env-ignore-getter-setter.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+Object.defineProperty(process.env, 'foo', {
+  value: 'foo1'
+});
```

Comment:
I am a bit surprised that this test passes. Is that intended as such?

Reply:
it's the same as 
```
{
  value: 'foo1',
  configurable: true,
  enumerable: true,
  writable: true
}
```

Reply:
That is super surprising... it behaves different than on any other object because on "regular" objects the output would be:

```
{
  value: 'foo1',
  configurable: false,
  enumerable: false,
  writable: false
}
```

The default for all descriptor properties is `false`. Can we please fix that as well by throwing the error for this case?

Reply:
ping @Himself65. Did you get a chance to look at @BridgeAR 's comment?

Reply:
Oh, I forgot. i will check later

Reply:
@BridgeAR if we do that, the previous code will throw an error

https://github.com/nodejs/node/blob/794d8be94e65a13cdbe12ea296637fe9e57e2878/test/parallel/test-worker-process-env.js#L41-L42

or we just output a warning?

Reply:
@Himself65 all of these "worked" before. AFAIK this is only limiting the possibilities to prevent confusion for users that expect the descriptor properties to be set as they did. Using `Object.defineProperty(obj, key, { value })` means the defaults are used for `enumerable`, `configurable` and `writable` and those are all set to `false`. If we do this, it should be consistent.

Reply:
Ping @Himself65

Reply:
resolved